projects/plutus/src/current_sense.c

```diff
@@ -0,0 +1,109 @@
+#include "current_sense.h"
+
+#include <string.h>
+
+#include "critical_section.h"
+#include "status.h"
+#include "wait.h"
+
+static void prv_calculate_current(int32_t *value, void *context) {
+  CurrentSenseStorage *storage = (CurrentSenseStorage *)context;
+  Status status = status_get();
```

Comment:
This won't work - status is updated globally.

Reply:
Neutered